### ISSUE TITLE
What is Structural Differnce between accurate/fast/default Mobilenet?

### ISSUE THREAD

**Issue by jone1222** [ildoonet/tf-pose-estimation#75]
Is there any structural difference between accurate/fast/default Mobilenet mentioned in this repo?

@ildoonet

**Comment by ildoonet** [comment]
Yes, basically, they have different feature map sizes in convolution layers. Also they have slightly different architectures. 

I am trying to find better architectures, I will add more if I find.
